src/cryptonote_basic/tx_extra.h

```diff
@@ -671,7 +671,7 @@ struct tx_extra_ethereum_service_node_leave_request {
     END_SERIALIZE()
 };
 
-struct tx_extra_ethereum_service_node_exit {
+struct tx_extra_ethereum_service_node_removal {
     uint8_t version = 0;
```

Comment:
Another note that now that exiting/liquidation is folded down into one event, for explainability at some point in the chain we should encode the reason the exit is coming through. Naturally people want reasons to why their node left the network/and or verify they didn't get penalised.